.github/ISSUE_TEMPLATE.md

```diff
@@ -16,4 +16,4 @@ e.g. http://example.com/2017/article
 - Tag 1
 - Tag 2
 
-_List all that apply from the [tags directory](../_tags/)._
+_List all that apply from the [tags directory](../tree/master/_tags)._
```

Comment:
This just kinda _moves_ the problem from the new issue to the repo.

Maybe just use a full URL instead of a relative path? We'd only need to change it if we decided to stop using GitHub.

Reply:
Here is the response I received from Shawna Jean (GitHub Staff): 

> Hi Derek,Relative links work in both places, but relative links depend on the URL of the current location. So, a relative link in a a new issue template can be viewed either at the location of the template in your repository: 

> https://github.com/cloudfour/pwastats/blob/master/.github/ISSUE_TEMPLATE.md

> Or in the new issue form itself:
> https://github.com/cloudfour/pwastats/issues/new

> And once submitted, is available at an issue's full URL: 
> https://github.com/cloudfour/pwastats/issues/144

> These URLs have different paths, so the relative URL will be based on a different location in each case. If you need to include a URL that will point to the same page from all three of these locations, a relative URL will not work. You'll need to use a full static URL.I hope this helps to clarify things!

> Best,Shawna